### PR TITLE
Valibot 0.30.0

### PR DESCRIPTION
**Pull request by arybitskiy**
**Please check if the PR fulfills these requirements**:

Codemod is not ready for 100%, but it covers all the cases mentioned in https://www.youtube.com/watch?v=hDJU-tXPVWE&ab_channel=FabianHiller
This is ast-grep + imperative approach

-   [ ] The commit message follows [our code of conduct](https://github.com/codemod-com/codemod-registry/blob/main/CODE_OF_CONDUCT.md) and [contributing guidelines](https://github.com/codemod-com/codemod-registry/blob/main/CONTRIBUTING.md)
-   [ ] Tests for the changes have been added (for bug fixes/features)
-   [ ] Docs have been added / updated (for bug fixes / features)

**Please include the following information in your pull request**:

-   **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

-   **What is the current behavior?** (You can also link to an open issue here)

-   **What is the new behavior (if this is a feature change)?**

-   **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

-   **Any other information (if needed)**:

### CODE REVIEW

File: packages/codemods/valibot/0.30.0/__testfixtures__/fixture1.output.ts
Comment (r4zendev) [inline packages/codemods/valibot/0.30.0/__testfixtures__/fixture1.output.ts:1]: output fixture

File: packages/codemods/valibot/0.30.0/README.md
Comment (r4zendev) [inline packages/codemods/valibot/0.30.0/README.md:1]: readme

File: packages/codemods/valibot/0.30.0/test/test.ts
Comment (r4zendev) [inline packages/codemods/valibot/0.30.0/test/test.ts:1]: can we add a test?

File: packages/codemods/valibot/0.30.0/README.md
Comment (mohab-sameh) [reply packages/codemods/valibot/0.30.0/README.md:1]: @r4zendev I can populate it as soon as the rest of the codemod is done. We can directly use description from the [migration guide](https://valibot.dev/guides/migrate-from-v0.30.0/).

**Review by r4zendev — APPROVED** [review]

**Pull request merged**